mutable_batch_lp/src/lib.rs

```
@@ -107,13 +133,35 @@ impl LinesConverter {
     ///     [`mutable_batch::writer::Error::TypeMismatch`]
     ///
     pub fn write_lp(&mut self, lines: &str) -> Result<()> {
+        // TODO: next PR will have this be configurable per env var.
+        let abort_on_line_error = true;
+
+        let mut errors: Vec<LineError> = Vec::with_capacity(MAXIMUM_RETURNED_ERRORS);
+        let mut add_error = |e: LineError| {
+            if errors.len() < MAXIMUM_RETURNED_ERRORS {
+                errors.push(e);
+            }
+            abort_on_line_error && errors.len() >= MAXIMUM_RETURNED_ERRORS
```

I'm not sure this is correct - perhaps an `if` would make it clearer?

This currently says:

"if abort_on_line_error == true AND more errors than the max allowed"

I think we're expecting one OR the other to stop parsing? Otherwise won't it evaluate all errors when `abort_on_line_error == false` (but not add them to the set) instead of returning early? 

I might be wrong, because it's not very clear going through the macro and loops :fire:

---

Already removed with refactoring.